PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/tree/TreeManager.java

```diff
@@ -52,31 +52,20 @@ public TreeManager(Pane graphPane, IPhylogeneticTreeNode root, Button zoomOutBut
    */
   private void initializeZoomOutEventHandler() {
     zoomOutButton.setOnAction(e -> {
-      if (isZooming) {
-        return;
-      }
-      if (currentRoot.getDataNode().hasParent()) {
```

Comment:
Maybe the ViewNode should implement some `hasParent()` method, which can internally call it on the DataNode, rather than chaining method calls? Decouples the classes more this way.

Not very well up to date with the GUI model though, so maybe that is not good design in this case.